src/main/webapp/app/exam/participate/events/exam-live-events-button.component.html

```diff
@@ -1,6 +1,6 @@
 @if (eventCount) {
     <button class="btn btn-md btn-info" (click)="openDialog($event)">
         <fa-icon [icon]="faBullhorn" />
-        <span class="d-none d-md-inline">{{ 'artemisApp.exam.events.button' | artemisTranslate }} ({{ eventCount }})</span>
+        <span class="d-none d-md-inline">({{ eventCount }})</span>
```

Comment:
Currently, the event count disappears on small screens, right? I would like to display it on small screens as well. 
That means simply removing the two classes.